nss-day-cohort-56/rare-server-team-2
# models and fixtures

Description

added models and fixtures for categories, comments, reactions, tags, posts, rare_user, user, and subscriptions. 

To Test: 
Run: 
python3 manage.py makemigrations app_api 
python3 manage.py migrate
python3 manage.py loaddata users tokens rare_users categories reactions tags posts comments subscriptions

then check sqlite explorer tables for data 

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings
